tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py3, pep8
+envlist = py35, py36, pep8
 
```

Comment:
You should remove basepython = python3 from [testenv] and add the following section:

[testenv:python3]
basepython = python3

Please also add "basepython = python3" to testenv:pep8.